### PR TITLE
fix(module): Add export for commonjs

### PR DESCRIPTION
**Pull request by thomasthiebaud**
Add commonjs export, so it is possible to require crossEnv using `require('cross-env')` (needed to
use in js script without babel)

#84

### CODE REVIEW

File: src/index.test.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import crossSpawnMock from 'cross-spawn'
 import crossEnv from '.'
 
+const commonJSCrossEnv = require('.')
```
Comment (kentcdodds) [inline src/index.test.js]: :+1:

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import {spawn} from 'cross-spawn'
 import commandConvert from './command'
 
+module.exports = crossEnv
 export default crossEnv
```
Comment (kentcdodds) [inline src/index.js]: Go ahead and remove this and just disable the rule for the offending line

Comment (thomasthiebaud) [inline src/index.js]: Yes sure, I do it now

Comment (thomasthiebaud) [inline src/index.js]: Do you prefer a new commit or a `push -f` ?

Comment (kentcdodds) [inline src/index.js]: Either way is fine, I'll do a squash and merge when I merge your pull request anyway

**Comment by kentcdodds** [comment]
Thanks!

**Pull request merged**